CHANGELOG.md

```diff
@@ -92,6 +92,11 @@
 - Adds Map and City view container - luc
 - Adds native unit test suite and CI - ash
 - Add FairMoreInfo screen - roop and javamonn
+- Adds shows list to Show View - ashley
```

Comment:
It should go under master right?

Reply:
I think these changes are pretty old and were under a different release, but I can move them to master.

Reply:
Ah ok, nvm then